python/src/adbc_example.py

```diff
@@ -0,0 +1,49 @@
+import sys
+from dataclasses import dataclass
+from urllib.parse import parse_qs, urlparse
+
+from adbc_driver_flightsql import DatabaseOptions
+from adbc_driver_flightsql.dbapi import connect
+
+
+@dataclass
+class ConnAttr:
+    host: str
+    params: dict
+    auth_header: str
+
+
+def parse_jdbc_uri(uri):
+    parsed = urlparse(uri)
+    params = {k.lower(): v[0] for k, v in parse_qs(parsed.query).items()}
+    return ConnAttr(
+        host=parsed.path.replace("arrow-flight-sql", "grpc")
+        if params.pop("useencryption") == "false"
+        else parsed.path.replace("arrow-flight-sql", "grpc+tls"),
+        params=params,
+        auth_header=f"Bearer {params.pop('token')}",
+    )
+
+
+def main(args):
+    conn_attr = parse_jdbc_uri(args[1])
+    with connect(
```

Comment:
I would wrap this in a function/library! this way it could be included in a notebook. It would be cool to have a simple function that conversts to a pandas df, 

```
import dbt.adbc.semantic_layer as sl

df = sl.query("""
    select * from {{semantic_layer.query(metrics=['cancellation_rate'])}}
""")